20_classes_lab/01.py

```diff
@@ -1,6 +1,22 @@
-"""
-The following code assumes a class
-named Summer exists. 
-Complete the class so correct result is printed
-"""
-
+class Summer(object):
+
+    def __init__(self):
+        self.items = []
+    def add(self, number):
+        self.items += [(number)]
+    def total(self):
+        return sum([i for i in self.items])
+
+s = Summer()
+t = Summer()
+
+s.add(10, 20)
+t.add(50)
+s.add(30)
+
+# should print 60
+print s.total()
+
+# should print 50
+print t.total()
+

```

Comment:
For accepting 2 or more values you might want to take a look at this SO question:
http://stackoverflow.com/questions/36901/what-does-double-star-and-star-do-for-python-parameters